README.md

```diff
@@ -261,13 +261,59 @@ const { _, isError } = useForm(/* ... */)
 isError /* true / false */
 ```
 
+3. Use `&nbsp;` to avoid height collapse of `<p>` when there is no messages
+
+#### Default message for form
+
+Use `defaultMessage` to define a placeholders for the form field validation error message. The default value is `''` , you can set it to `u00A0` , which will be escaped to `&nbsp` during rendering , to avoid the height collapse problem of '<p>' when there is no messages.  
```

Comment:
```suggestion
Use `defaultMessage` to define a placeholders for the form field validation error message. The default value is `''`, you can set it to `u00A0`, which will be escaped to `&nbsp` during rendering, to avoid the height collapse problem of '<p>' when there is no messages.  
```

还是有三个多余的空格哦

Reply:
不过不清楚是不是在你的规范里用 ` 括起来的应该有自己的空格？

Reply:
是的。。。我想或许需要一个项目中的 md-lint 来帮助校验这些，目前貌似是没有的

Reply:
在最初选择的时候有考虑过使用 `prettier` 吗，或者说为什么推荐使用 `dprint` 而不是其他的，之前我还没有接触到过 `dprint` 这个插件，大部分项目中貌似都会选择使用 `prettier` 来做格式化

Reply:
例如在我的项目中会选择 eslint + prettier + stylelint，然后结合 .vscode 配置保存自动格式化以及修复错误，开发时使用起来感觉会比较方便

Reply:
谢谢解答，学习了，抱歉我在先前并不知道这样的规则。

@LittleSound 可以看看怎么样配置一下，方便开发流程和体验

Reply:
项目里应该是有 `prettier` 和 `eslint` 可以一同帮忙格式化的，但是对于 Markdown 文件而言还是欠缺了。